frigate/__main__.py

```diff
@@ -18,7 +16,8 @@ def main() -> None:
     )
 
     threading.current_thread().name = "frigate"
-    cli.show_server_banner = lambda *x: None
+    # TODO: Rui. Do we need this?
```

Comment:
this should be removable, I believe it was just for having the flask banner not show in the logs

Reply:
Fixed